src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -628,6 +633,124 @@ available on the [Fuchsia devsite].
 Running the Rust test suite on Fuchsia is [not currently supported], but work is
 underway to enable it.
 
+## Debugging
+
+### `zxdb`
+
+Debugging components running on a Fuchsia emulator can be done using the
+console-mode debugger: [zxdb]. We will demonstrate attaching necessary symbol
+paths to debug our `hello-fuchsia` component.
+
+### Attaching `zxdb`
+
+In a separate terminal, issue the following command from our `hello_fuchsia`
+directory to launch `zxdb`:
+
+**In separate terminal**
+```sh
+${SDK_PATH}/tools/${ARCH}/ffx debug connect -- \
+    --symbol-path $(pwd)/target/x86_64-fuchsia/debug
```

Comment:
Is the absolute path required? We don't use them elsewhere in the guide

Reply:
Good point -- tested and absolute path is not necessary. Removing